tasklib/tests.py

```diff
@@ -327,9 +330,9 @@ def test_add_completed_task(self):
 
     def test_add_multiple_completed_tasks(self):
         t1 = Task(self.tw, description="test1", status="completed",
-                 end=datetime.datetime.now())
+                  end=datetime.datetime.now())
```

Comment:
I know it's verbose, but putting each argument on its own line if they won't all fit on a single one makes things really clear, in my opinion. It also means you can add/remove arguments while keeping a very clear diff in Git.

```python
t1 = Task(
    self.tw,
    description="test1",
    status="completed",
    end=datetime.datetime.now(),
)
```

Reply:
Fixed in 8f4fabf